dozer-sql/src/pipeline/builder.rs

```diff
@@ -117,6 +118,7 @@ fn query_to_pipeline(
     query_ctx: &mut QueryContext,
     stateful: bool,
     pipeline_idx: usize,
+    is_top_select: &mut bool,
```

Comment:
`is_top_select: bool` should be enough? This doesn't look like a input output parameter.